frontend/workflows/projectCatalog/src/catalog/index.tsx

```diff
@@ -103,13 +113,36 @@ const Catalog: React.FC<WorkflowProps> = ({ heading }) => {
           />
         </div>
       </Paper>
-      <div style={{ marginBottom: "16px", marginTop: "32px" }}>
+      <div
+        style={{
+          display: "flex",
+          justifyContent: "space-between",
+          marginBottom: "16px",
+          marginTop: "32px",
+        }}
+      >
         <Typography variant="h3">My Projects</Typography>
+        <IconButton
+          variant="neutral"
+          onClick={() => {
+            clearProjects();
+            dispatch({ type: "HYDRATE_START" });
+            getProjects(
+              projects => dispatch({ type: "HYDRATE_END", payload: { result: projects } }),
+              setError
+            );
+          }}
+        >
+          <RestoreIcon />
+        </IconButton>
       </div>
       <Grid container direction="row" spacing={5}>
         {state.projects.map(p => (
           <Grid item onClick={() => navigateToProject(p)}>
-            <ProjectCard project={p} onRemove={() => triggerProjectRemove(p)} />
+            <ProjectCard
+              project={p}
+              onRemove={state.projects.length !== 1 ? () => triggerProjectRemove(p) : undefined}
```

Comment:
I feel like this behavior is going to be a little weird if we're not going to put up a message or something.

Might be better to just have a default div that shows up if there's no projects defined:

'No Projects Currently Added, Initialize Projects?' or something which could use the new reset features.